Formula/wxpython.rb

```diff
@@ -30,36 +28,36 @@ class Wxpython < Formula
   end
 
   resource "Pillow" do
-    url "https://files.pythonhosted.org/packages/5b/bb/cdc8086db1f15d0664dd22a62c69613cdc00f1dd430b5b19df1bea83f2a3/Pillow-6.2.1.tar.gz"
-    sha256 "bf4e972a88f8841d8fdc6db1a75e0f8d763e66e3754b03006cbc3854d89f1cb1"
+    url "https://files.pythonhosted.org/packages/3e/02/b09732ca4b14405ff159c470a612979acfc6e8645dc32f83ea0129709f7a/Pillow-7.2.0.tar.gz"
+    sha256 "97f9e7953a77d5a70f49b9a48da7776dc51e9b738151b22dacf101641594a626"
   end
 
   resource "six" do
-    url "https://files.pythonhosted.org/packages/94/3e/edcf6fef41d89187df7e38e868b2dd2182677922b600e880baad7749c865/six-1.13.0.tar.gz"
-    sha256 "30f610279e8b2578cab6db20741130331735c781b56053c59c4076da27f06b66"
-  end
-
-  # Fix build for 10.15 SDK (WebKit errors).
-  # Fixed in 4.1.x.
-  patch do
-    url "https://raw.githubusercontent.com/Homebrew/formula-patches/20da70f67040d40f8420bd7d543c875739261e24/wxpython/10.15-sdk.patch"
-    sha256 "c54f2bb97532f483219c63a3c9e463d4aeda759bee2fff1f07820f3c752f68f1"
+    url "https://files.pythonhosted.org/packages/6b/34/415834bfdafca3c5f451532e8a8d9ba89a21c9743a0c59fbd0205c7f9426/six-1.15.0.tar.gz"
+    sha256 "30639c035cdb23534cd4aa2dd52c3bf48f06e5f4a941509c8bafd8ce11080259"
   end
 
   def install
-    # Fix build of included wxwidgets
-    # see https://github.com/wxWidgets/Phoenix/issues/1247
-    inreplace "buildtools/build_wxwidgets.py",
-              /^( +)(wxpy_configure_opts.append\("--disable-qtkit"\))/,
-              "\\1\\2\n\\1wxpy_configure_opts.append(\"--disable-precomp-headers\")"
+    # Fix build of included wxwidgets:
```

Comment:
Don't we have a formula for that?

Reply:
Yes, although I'm not sure if we can trick `wxpython` into integrating with an existing `wxmac` installation (I've yet to find an option to avoid using bundled `wxwidgets`). I shall experiment a bit.

Reply:
After further review, I've found that there is a way to build `wxpython` with an existing `wxmac` installation.

However, we are shipping `wxmac` 3.0.5 while `wxpython` bundles (and depends on features/files that are only present in) 3.1.4. So the `wxpython` build fails when trying to build against `wxmac` because there are certain header files, etc. missing.

For context, [the upgrade for `wxmac` was blocked](https://github.com/Homebrew/homebrew-core/pull/58413) due to the entire 3.1.x branch currently being denoted as a development branch while 3.0.5 remains the current stable.

I made a new formula out of the updated `wxmac` PR linked there and made `wxpython` depend upon it, and it has passed build and test without any issues. So I think the real blocker here really is just that we can't upgrade `wxmac`.